python/interpret_community/explanation/explanation.py

```diff
@@ -1148,25 +1148,41 @@ def eval_data(self):
         :return: The dataset or dataset ID.
         :rtype: list or str
         """
-        return self._eval_data
+        return self._convert_to_list(self._eval_data)
 
     @property
     def eval_y_predicted(self):
         """Get predicted ys for the evaluation data.
 
         :return: The predicted ys for the evaluation data.
-        :rtype: np.array
+        :rtype: list
```

Comment:
do we need to specify type here like list[int | float] or not?

Reply:
will add a generic